birdhouse/config/jupyterhub/custom_templates/login.html

```diff
@@ -1,8 +1,21 @@
-{% extends "templates/login.html" %}
-{% set announcement_login = '
+{% extends "templates/login.html" %} {% set announcement_login = '
 <p><strong>Public login:</strong> public/public</p>
-<p>Given this public nature, anyone can tamper with your notebooks so please <strong>export your valuable notebooks elsewhere</strong> if you want to preverve them.</p>
-<p>Your <strong>notebooks will persist</strong> accross reboots <strong>only if</strong> saved under <strong>home (/notebook_dir)</strong>.</p>
-<p>Please <strong>be considerate</strong> with the amount of <strong>disk space usage</strong> on this Jupyter instance.</p>
-<p>This Jupyter instance can restart every day.  <strong>Long running processes will be killed without notice.</strong></p>
+<p>
+  Given this public nature, anyone can tamper with your notebooks so please
```

Comment:
I think we can remove this "public nature" paragraph since your PR effectively make each user workspace private to him only.

Reply:
So maybe add a note on how to ask for an account? If so, could you write something like 'contact Ouranos for an account'?

Reply:
ah, there is a SUPPORT_EMAIL variable in env.local

Reply:
Oh wait, actually we still need to keep that public nature paragraph for the public/public demo user !  

But we can add "contact us for account" like you said if user want a personalized private account with private workspace.  We can use the private workspace as a selling point for user to make the effort to contact us.